src/components/base/b-list/modules/dom.ts

```diff
@@ -0,0 +1,21 @@
+/*!
+ * V4Fire Client Core
+ * https://github.com/V4Fire/Client
+ *
+ * Released under the MIT license
+ * https://github.com/V4Fire/Client/blob/master/LICENSE
+ */
+
+import type Block from 'components/friends/block/class';
+
+export function setActive(block: Block | undefined, el: Element, status: boolean): void {
```

Comment:
```suggestion
export function setActive(block: Nullable<Block>, el: Element, status: boolean): void {
```

И описания нет. Плюс, тебе не кажется, что т.к. всего 2 функции в modules, то можно их водном файлике helpers.ts оставить, чтобы не было слишком высокой фрагментации